security_test.go

```diff
@@ -304,7 +300,7 @@ var _ = Describe("Security", func() {
 
 				_, err := container.Run(garden.ProcessSpec{
 					User: "alice",
-					Path: "ifconfig", // ifconfig is only available in /sbin
+					Path: "hello-world", // hello-world is only available in /sbin
```

Comment:
What caused this to change? Wouldn't `ip` be a better choice?

Reply:
I think this test is not really testing anything, but I decided to leave it as is. The test is testing for a binary that is on the path, in this case, it picks `/sbin`. busbox images had no binary under `/sbin`, so I added [`hello-world` to fulfill this test](https://github.com/cloudfoundry/wg-app-platform-runtime-ci/blob/main/garden-runc-release/dockerfiles/garden-rootfs/Dockerfile#L75-L76), but I think we can remove this test.